### PR TITLE
[2.13.x] DDF-4570 Fixes installer overriding solr password

### PR DESCRIPTION
**Pull request by vinamartin**
#### What does this PR do?
Fixes an issue where the installer would overwrite the solr password after it had already been set earlier on in the install process. 

#### Who is reviewing it? 
@ahoffer @Kjames5269 @jhunzik 

#### Ask 2 committers to review/merge the PR and tag them here.
@andrewkfiedler
@emmberk

#### How should this be tested?
Run through the installer and hit restart now at the end.
Open DDF and verify that Intrigue starts up. 
Look at custom.system.properties and verify that the solr.password is some encrypted password.

#### Any background context you want to provide?
This PR (https://github.com/codice/ddf/pull/4429) introduced the issue by frontloading the read of the system properties file which makes it fall out of sync after the solr properties are updated.
I've refactored it to read every time these methods are called to ensure it doesn't fall out of sync.

#### What are the relevant tickets?
Fixes: #4570

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/SystemPropertiesAdmin.java [hunk 1]
```diff
@@ -114,11 +114,6 @@ public SystemPropertiesAdmin(GuestClaimsHandlerExt guestClaimsHandlerExt)
     super(SystemPropertiesAdminMBean.class);
     this.guestClaimsHandlerExt = guestClaimsHandlerExt;
     configureMBean();
-    try {
-      systemDotProperties = new Properties(systemPropertiesFile);
```
Comment (emmberk) [inline platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/SystemPropertiesAdmin.java]: We want to remove the `private Properties systemDotProperties;` class field, right?

Comment (vinamartin) [reply platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/SystemPropertiesAdmin.java]: Ah good catch, there was a conflict on that line so I must have missed it!

**Comment by vinamartin** [comment]
build now

**Comment by austinsteffes** [comment]
hero successful ✅

**Pull request merged**